Automattic/woocommerce-payments
# Format customer currency based on the customer's country

Fixes #2245 

#### Changes proposed in this Pull Request

* Use country-specific formatting when building currency format from locale-info
* Add get_user_locale_country method to Utils
* Use customer country in get_currency_format
* Add \_admin\_ to multi-currency register/enqueue scripts
* Update mini cart when the billing country is updated on legacy checkout

Now that we're relying on the user's locale this gets a bit simpler and all of the ~strikethrough~ removed in this edit is not relevant anymore.









#### Testing instructions




You'll need a way to change your locale to test this. You could change your site's default locale under Settings > Site Language.

*Legacy checkout*

* Enable `EUR` under the multi-currency settings
* Change the locale to `es_ES`
* Go to the checkout page and make sure that the currencies in the order summary and mini-cart are in the right format. E.g. 2.354,55€
* Navigate the Shop page and make sure the prices are in the same format
* Change the locale to `nl_NL`
* Go to the checkout page and make sure that the currencies in the order summary and mini-cart are in the right format. E.g. €2,354.55
* Navigate the Shop page and make sure the prices are in the same format

*Blocks checkout*

* Change the locale to `es_ES`
* Go to the blocks checkout and make sure that the currencies in the order summary and mini-cart are in the right format (with the exception of symbol placement). E.g. €2.354,55
* Change the locale to `nl_NL`
* Go to the blocks checkout and make sure that the currencies in the order summary and mini-cart are in the right format. E.g. €2,354.55

-------------------
